samples/datastorage/sample-json-device.yaml

```diff
@@ -0,0 +1,61 @@
+name: "7e841a12-a083-48fe-8040-83815e0b7410"
+manufacturer: "Intel Corp."
```

Comment:
@suresh-lc If this is a sample, what about anonymizing the manufacturer name for the device? Like _Vendor A_

Reply:
OK sure